__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app.js')
+const request = require('supertest')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const { articleData, topicData, userData, commentData } = require('../db/data/test-data/index.js')
+
+// require('jest-sorted')
+
+beforeEach(() => {
+    return seed({ articleData, topicData, userData, commentData })
+})
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET:/api/topics', () => {
+    test('GET: 200 - sends a list of topics to the client', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
+                console.log(body.topics)
+                expect(body.topics.forEach(topic => {
+                    expect(typeof topic.description).toBe('string')
+                    expect(typeof topic.slug).toBe('string')
+                }))
+            })
+    })
```

Comment:
Great test. As it's gonna reseed before each test, so the array won't change, thus, it would be better to check the length of the array as well. And to be more specific what the datatype of the values of slug and description. (object.containing would help)